### PR TITLE
Add admin docs for holding a node

### PR DESCRIPTION
**Pull request by ConsoleCatzirl**
Add admin docs for holding a node

### CODE REVIEW

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/lore/51/1486062357.79/lore : FAILURE in 28s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/lore/51/1486062398.18/lore : FAILURE in 10s

File: admins/hold-node.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| ID  | Provider | AZ   | Label         | Target | Manager | Hostname                  | NodeName                  | Server ID                            | IP            | State | Age         | Comment |
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+---------------+-------+-------------+---------+
+| 248 | cicloud  | None | ubuntu-xenial | zuul   | zuul    | ubuntu-xenial-cicloud-248 | ubuntu-xenial-cicloud-248 | a7a18e16-0284-4122-930e-81e3af03af82 | 192.168.3.241 | ready | 00:00:21:21 | None    |
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+---------------+-------+-------------+---------+
+```
+
+## Holding a Nodepool Node
+
+If you are targetting a specific node currently running a test, lookup the ID based on the hostname provided at the begginning of the test output. You will need to be quick in order to hold the node before the test completes and the node is subsequently deleted.
```
Comment (eggshell) [inline admins/hold-node.md]: Extra t in targeting.

File: admins/hold-node.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+If you are targetting a generic node, choose any ID for a node in the `ready` state.
```
Comment (eggshell) [inline admins/hold-node.md]: Same as above.

**Review by eggshell — DISMISSED** [review]
Looks great, just have two typos to fix and I'll merge.

File: admins/hold-node.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Holding a Node
+
```
Comment (mlangbehn) [inline admins/hold-node.md:2]: Can we add a table of contents to make skimming easier?

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/lore/51/1486064084.62/lore : FAILURE in 24s

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.com/check_github/BonnyCI/lore/51/1486067111.16/lore : FAILURE in 21s

**Comment by mlangbehn** [comment]
recheck

**Review by eggshell — APPROVED** [review]

**Review by mlangbehn — APPROVED** [review]
Thanks!

**Pull request merged**